include/seqan3/alphabet/quality/composition.hpp

```diff
@@ -44,13 +44,13 @@
 #include <string>
 #include <utility>
 
-#include <seqan3/alphabet/composition.hpp>
+#include <seqan3/alphabet/composition/cartesian_composition.hpp>
 #include <seqan3/alphabet/nucleotide/concept.hpp>
 
 namespace seqan3
 {
 
-/*!\brief An alphabet_composition that joins a nucleotide alphabet with a quality alphabet.
+/*!\brief An cartesian_composition that joins a nucleotide alphabet with a quality alphabet.
```

Comment:
"A" instead of "An", and please add `seqan3::` or doxygen won't generate a link.